sway/config/input.c

```diff
@@ -90,6 +90,16 @@ void merge_input_config(struct input_config *dst, struct input_config *src) {
 	}
 }
 
+struct input_config *copy_input_config(struct input_config *ic) {
+	struct input_config *copy = calloc(1, sizeof(struct input_config));
+	if (copy == NULL) {
+		wlr_log(L_ERROR, "could not allocate input config");
+		return NULL;
+	}
+	merge_input_config(copy, ic);
```

Comment:
merge should be the other way around?

Reply:
It looks correct to me. The destination should be `copy` and the source should be `ic`.

Reply:
doh, you're right, I was convinced it was the other way around for some reason... Even opened its implementation and didn't read what I had under my eyes, I need more sleep -_-